translate/yaml.go

```diff
@@ -0,0 +1,90 @@
+package translate
+
+import (
+	"fmt"
+
+	"github.com/arr-ai/arrai/rel"
+	"gopkg.in/yaml.v2"
+)
+
+// YAMLToArrai translates an object unmarshalled from yaml into an array value
+//
+// translation follows the rules
+//
+//     object -> {|@,@item|, |key,val|, ...}
```

Comment:
I think the mapping behaviour could be really useful in a standalone doc in the docs folder
Also, we should document in https://github.com/arr-ai/arrai/blob/master/docs/intro.md that arrai now supports YAML and JSON expressions and imports

Reply:
@tiagralee There is a documentation here for the JSON https://github.com/arr-ai/arrai/blob/master/docs/std-encoding-json.md, it should be used to avoid duplicate.